### PR TITLE
Add getLayers and setLayers functions to ol.format.WMSGetFeatureInfo

### PR DESCRIPTION
**Pull request by fredj**
Same idea as #7410 but for `ol.format.WMSGetFeatureInfo`

### CODE REVIEW

File: src/ol/format/wmsgetfeatureinfo.js [hunk 1]
```diff
@@ -64,6 +64,22 @@ ol.format.WMSGetFeatureInfo.layerIdentifier_ = '_layer';
 
 
 /**
+ * @return {Array.<string>} layers
+ */
+ol.format.WMSGetFeatureInfo.prototype.getlayers = function() {
```
Comment (ahocevar) [inline src/ol/format/wmsgetfeatureinfo.js]: Shouldn't this be camel case (`getLayers`)?

File: src/ol/format/wmsgetfeatureinfo.js [hunk 2]
```diff
@@ -64,6 +64,22 @@ ol.format.WMSGetFeatureInfo.layerIdentifier_ = '_layer';
...
+  return this.layers_;
+};
+
+
+/**
+ * @param {Array.<string>} layers Layers to parse.
+ */
+ol.format.WMSGetFeatureInfo.prototype.setlayers = function(layers) {
```
Comment (ahocevar) [inline src/ol/format/wmsgetfeatureinfo.js]: Shouldn't this be camel case (`setLayers`)?

File: src/ol/format/wmsgetfeatureinfo.js (hunk 1 above)
Comment (fredj) [reply src/ol/format/wmsgetfeatureinfo.js]: ouch. yes, changed

**Comment by fredj** [comment]
I don't need the `@api` annotations because OpenLayers is compiled with my application but I can add them.

**Comment by ahocevar** [comment]
> I don't need the @api annotations because OpenLayers is compiled with my application but I can add them.

I'll leave it up to you. Do you want to have this documented and maintain it? Then add the annotation. If you don't want anyone to know about it and use it, then leave it as-is.

**Comment by fredj** [comment]
Thanks for the review @ahocevar

**Pull request merged**